client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -31,23 +32,39 @@ export function PatternPreview( {
 	}
 
 	return (
-		<div
-			className={ classNames( 'pattern-preview', {
-				'pattern-preview_category-gallery': isCategoryPreview,
-				'is-loading': ! renderedPattern,
-			} ) }
+		<ResizableBox
```

Comment:
The `ResizableBox` wrapper seems to mess up the pattern previews in the category links on the `/patterns` page. We should do something about that.

![image](https://github.com/Automattic/wp-calypso/assets/1101677/8b448687-eeca-4be6-81d8-66265fc97cfe)

Reply:
We should also disable the resizing entirely in grid view. 
![grid view](https://github.com/Automattic/wp-calypso/assets/1101677/fc8683a1-b175-471a-9d5b-6c1e066275a0)

Reply:
> We should also disable the resizing entirely in grid view.

Done.

> The ResizableBox wrapper seems to mess up the pattern previews in the category links on the /patterns page. We should do something about that.

It looks like it's not needed in that case either. So I have added an `isResizable` prop to actively indicate intent to resize